pkg/config/config_test.go

```diff
@@ -512,6 +514,9 @@ image_copy_tmp_dir="storage"`
 			// When
 			config, err := NewConfig("")
 			// Then
+			if runtime.GOOS != "linux" {
+				Skip(fmt.Sprintf("capabilities not supported on %s", runtime.GOOS))
```

Comment:
Can you move the `Skip` above?  This will spare a couple of cycles when running on non-Linux machines.

Reply:
I would move it above the `// When`. Currently, we'd be calling `NewConfig("")` but ignore the return values.